src/molecules/Menu.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import Text from '../atoms/Text'
+
+const StyledMenu = styled.ul`
+  text-decoration: none;
+  padding: 0;
+  width: 100%;
+`
+const StyledList = styled.li`
+  list-style: none;
+  display: inline-block;
+  padding: 0.8rem;
+  margin: 0.5rem 0.8rem;
+
+  &:hover {
+    color: red;
+  }
+`
+
+export const menu = [
+  {
+    value: 'OVERVIEW',
```

Comment:
Don't write text in capital letters. You can always use `text-transform: uppercase` if you need them in capital visually. Keep text in normal case.

```diff
- value: 'OVERVIEW'
+ value: 'Overview'
```